### ISSUE TITLE
Standarize font between default icons and custom icons

### ISSUE THREAD

**Issue by Pabl0cks** [DynOS-95/DynOS-95#21]
Currently we have W95FA font embedded in the default icons, and we're creating new custom dapp icons with default font.

We have to think in the better way to have a consistent font for both of them.

**Comment by andrealbiac** [comment]
Right, cause we can't 'load' the font in the code and also use it as the default font?

Maybe a Google font would be a good alternative if it's easier to implement for us?
